lib/cmd_line.dart

```diff
@@ -26,34 +25,44 @@ import 'package:gltf/src/errors.dart';
 import 'package:gltf/src/utils.dart';
 import 'package:isolate/load_balancer.dart';
 import 'package:isolate/isolate_runner.dart';
-import 'package:path/path.dart' as path;
+import 'package:path/path.dart' as p;
```

Comment:
Should this be more than a one-letter name?

Reply:
It's the convention for using that package.